LuaMenu/widgets/snd_music_lite.lua

```diff
@@ -154,13 +154,13 @@ function widget:Initialize()
 		return
 	end
 
-	OPEN_TRACK_NAME = WG.Chobby.Configuration.gameConfig.openTrack
-	if OPEN_TRACK_NAME ~= nil then
-		if OPEN_TRACK_NAME == nil or OPEN_TRACK_NAME == '' then
-			OPEN_TRACK_NAME = randomTrackList[math.random(#randomTrackList)]
+	openTrack = WG.Chobby.Configuration.gameConfig.openTrack
+	if openTrack ~= nil then
+		if openTrack == nil or openTrack == '' then
+			openTrack = randomTrackList[math.random(#randomTrackList)]
 		end
 	end
-	if OPEN_TRACK_NAME == nil or OPEN_TRACK_NAME == '' then
+	if openTrack == nil or openTrack == '' then
```

Comment:
Why can't this widget run if there is no open track? It would just mean there is silence until the first game is over (at which point a track will play).

Reply:
Shouldn't this be 

`if not openTrack then'